### PR TITLE
Add the training section to the navbar and the NASA TOPS event page

### PR DESCRIPTION
**Pull request by yuko29**
I add the "training" section to the navbar, and place the page for upcoming NASA Open Science Skills Training there.
I also move the workshop page to the training section.

### CODE REVIEW

File: content/pages/training/2024/08-nasatops.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+============================================================
```
Comment (yungyuc) [inline content/pages/training/2024/08-nasatops.rst:1]: `workshop/` may be a more inclusive name for the directory, while I am ok with `training/`.

**Review by yungyuc — COMMENTED** [review]
@chestercheng @sucitw @YenPeiChen07 could you please help review?

**Comment by yuko29** [comment]
I updated the content to make the event information clearer, including the dates, times, location, and registration link.

**Comment by YenPeiChen07** [comment]
It looks good to me.

**Pull request merged**